src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.scss

```diff
@@ -0,0 +1,3 @@
+.bc-legend-help {
+    font-size: small;
+}
```

Comment:
The help text below legend is taking its parents' font css, hence need to enforce this. 
Without this it will look like:
![image](https://github.com/ceph/ceph/assets/25664409/7cc077b7-456e-4d1a-9ab3-aecba0d28eaf)